apps/codeforafrica/src/lib/api.netlify-cms/getProjects.js

```diff
@@ -43,6 +44,15 @@ export default function getProjects(fields) {
         list: donors?.filter((donor) => project.donors.includes(donor.id)),
       };
     }
+
+    if (fields.includes("team")) {
+      const team = getTeam();
+      project.team = {
+        title: "Team",
+        list: team?.filter((member) => project.team.includes(member.id)),
+      };
+    }
```

Comment:
```suggestion
    if (project.team?.length) {
      const members = getTeam();
      project.team = {
        title: "Team",
        list: project.team.map((id) => members.find((m) => m.id === id)) ,
      };
    }
```

It's best to keep the order of how the members were defined in project rather than default order in teams (may be we should do this for all project related fields such as donors, badges, etc.?)